AsyncDisplayKit/ASImageNode.mm

```diff
@@ -123,11 +123,8 @@ - (void)setImage:(UIImage *)image
   if (!ASObjectIsEqual(_image, image)) {
     _image = image;
 
-    ASDN::MutexUnlocker u(_imageLock);
```

Comment:
This is so awesome! Presumably we want to keep this unlocked call, however, or we will be holding the lock for the next two methods which is not required?